### PR TITLE
Fix documentation referencing Dev Preview limitations and includes Cron Jobs for Online

### PR DESCRIPTION
**Pull request by wgordon17**
* Dev Preview had some PV limitations that are no longer in effect for Starter/Pro
* Pro tier is able to use Cron Jobs, so the documentation should be included with a notice for Starter users

Signed-off-by: Will Gordon <wgordon@redhat.com>

### CODE REVIEW

**Comment by wgordon17** [comment]
Apologies for the additional edits in files, my text editor automatically removes trailing spaces.

**Comment by wgordon17** [comment]
@adellape Care to review for the online branch?

**Comment by wgordon17** [comment]
@ahardin-rh Would you mind reviewing this?

File: dev_guide/cron_jobs.adoc [hunk 1]
```diff
@@ -27,6 +27,15 @@ As of {product-title} 3.3.1, Cron Jobs is a feature in Technology Preview.
 ====
 endif::[]
 
+ifdef::openshift-online[]
+[IMPORTANT]
+====
+Cron Jobs are only available to _Pro tier_. For more information about the
```
Comment (ahardin-rh) [inline dev_guide/cron_jobs.adoc]: available for

Comment (ahardin-rh) [inline dev_guide/cron_jobs.adoc]: Instead of Pro tier I would say OpenShift Online Pro

Comment (wgordon17) [inline dev_guide/cron_jobs.adoc]: 🤦🏻‍♂️ Thanks!

**Comment by ahardin-rh** [comment]
@wgordon17 Thanks! Mergin'! :shipit:

**Comment by ahardin-rh** [comment]
No revision history needed

**Pull request merged**